README.md

```diff
@@ -114,8 +114,8 @@ COPY --from=builder /usr/share/python3/app /usr/share/python3/app
 # Install the required library packages
 RUN xargs -ra /usr/share/python3/app/pkgdeps.txt apt-install
 
-# Create a symlink to the target binary (just for convenience)
-RUN ln -snf /usr/share/python3/app/bin/ipython /usr/bin/
+# Add entrypoints from the venv to PATH (e.g. 'pip', 'python', just for convenience)
+ENV PATH="/usr/share/python3/app/bin:${PATH}"
```

Comment:
Yes it's good but in this case the `python3` starts running from `/usr/share/python3/app/bin` it's not safe default IMHO.

Might be add a commented line with this alternative way and describe it.

Reply:
```suggestion
ENV PATH="${PATH}:/usr/share/python3/app/bin"
```

Is it safer like this?

Reply:
```console
$ which python
/usr/share/python3/app/bin/python
$ which python3
/usr/bin/python3
```

Reply:
Inconsistent with python2 behaviour